vault/dendron.release.changelog.md

```diff
@@ -2,11 +2,25 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1636140849625
+updated: 1636480201973
 created: 1601508213606
 date: '2022-01-01'
 ---
 
+## 0.68.0
+
+### Features
+- feat(schemas): adding new command - create schema from hierarchy ([[docs|dendron.topic.schema.create-from-hierarchy]]) (#1673) @nickolay
+- feat(views): native dendron preview ([[docs|]]) @kevin
```

Comment:
no documentation since underlying feature hasn't changed. would just copy the updates from the scratch note under here